### ISSUE TITLE
Loss unnormal when changing the image size?

### ISSUE THREAD

**Issue by feiyuhuahuo** [dbolya/yolact#182]
I trained on one RTX2070 with batch_size=6 and with different image sizes: 400, 512, 550 with the code modified by myself (I did not change the code's logic I think. I just simplified the code, changed nothing, but maybe I missed something).
This is your original code for image size: 550:
![550_origin](https://user-images.githubusercontent.com/32631344/67059342-33019280-f18b-11e9-8f17-b9d0cd72d272.png):
image size: 512 (just changed max_size to 512, did nothing to anchor scale)
![512_origin](https://user-images.githubusercontent.com/32631344/67059379-5fb5aa00-f18b-11e9-9d58-6f2161d76b5b.png)
image size:400 (yolact_im400_config):
![400_origin](https://user-images.githubusercontent.com/32631344/67059400-71974d00-f18b-11e9-83cc-5eacac86f611.png)
Then is on my modified code:
image size: 550:
![550_mine](https://user-images.githubusercontent.com/32631344/67059419-8c69c180-f18b-11e9-8957-1ee0fa03e6d7.png)
image size: 512:
![512_mine](https://user-images.githubusercontent.com/32631344/67059428-955a9300-f18b-11e9-83ef-f11d80a991e0.png)
image size 400:
![400_mine](https://user-images.githubusercontent.com/32631344/67059434-9db2ce00-f18b-11e9-9d53-29b4977bd883.png)

Seems when the image size is 550, my code training normally (and I do training with 550 and get 32.1 bbox mAP, 29.6 mask mAP finally). But when is 512 and 400, something is wrong. So any suggestions for debugging? I don't know how to deal with this.
By the way, the model forward progress is normal, which means when testing mAP with image size=550 using the .pth file you provide, the result is exactly the same.

**Comment by dbolya** [comment]
Do you know what exactly you changed (`git diff` perhaps)? It's obviously hard to say what went wrong, but I'd first take a look at any changes you made to the loss function or the prior box creation.

**Comment by feiyuhuahuo** [comment]
OK, I fixed this. I created anchors wrongly.

**Comment by Ira236** [comment]
![grafik](https://user-images.githubusercontent.com/54095782/71768350-993dbc00-2f15-11ea-872a-a1c798a9b6b8.png)

I think I have the same problem, any ideas?
